microceph/ceph/config.go

```diff
@@ -222,6 +233,10 @@ func setConfigItem(c types.Config) error {
 		return err
 	}
 
+	if config.Permission == ClusterConfigEXPRW {
+		return fmt.Errorf("WARNING: set operation on experimental config (%s)", c.Key)
```

Comment:
I would advise against doing this. The pattern `if err != nil` is way too common in Go, and overloading errors for warnings seems like a footgun to me. Instead, I would simply log the warnings here instead of down below.

Reply:
The problem with logging is that the end user will not know about it until they check the logs. An acceptable (rather better) alternative is to require `--experimental` flag for such configs.